Team_Roster/app/routes.py

```diff
@@ -16,6 +18,11 @@ def roster(teamId):
     year =request.args.get("year")
     return ShowRoster(teamId,year)
 
+@app.route('/<teamId>/depthChart', methods=['GET'])
```

Comment:
why did you add a route here? The pitching data can just go in the roster page, using the following files:
```
roster.py
roster.html
```